app/Http/Controllers/Service/Admin/VouchersController.php

```diff
@@ -53,9 +53,12 @@ public function storeBatch(AdminNewVoucherRequest $request)
         $sponsor = Sponsor::findOrFail($input['sponsor_id']);
         $now_time = Carbon::now();
         $maxStep = 1000;
+        $start = $input['start-serial'];
+        $end = $input['end-serial'];
+        $pad = strlen($end);
 
-        // Calulate the number of codes we need
-        $numCodes = $input['end'] - $input['start'];
+        // Calculate the number of codes we need
+        $numCodes = $start - $end;
```

Comment:
This looks like it might have been accidentally flipped

Reply:
yep!